src/bin/crawl

```diff
@@ -15,26 +15,32 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-# Usage: crawl [-i|--index] [-D "key=value"] [-w|--wait] [-s <SeedDir>] [-sm <SitemapDir>] <CrawlDir> <NumRounds>
-#    -i|--index      Indexes crawl results into a configured indexer
-#    -w|--wait       NUMBER[SUFFIX] Time to wait before generating a new segment when no URLs
-#                    are scheduled for fetching. Suffix can be: s for second,
-#                    m for minute, h for hour and d for day. If no suffix is
-#                    specified second is used by default.
-#    -D              A Java property to pass to Nutch calls
-#    -s              Path to seeds file(s)
-#    -sm             Path to sitemap URL file(s)
-#    CrawlDir        Directory where the crawl/link/segments dirs are saved
-#    NumRounds       The number of rounds to run this crawl for
+# Usage: crawl [options] <crawl_dir> <num_rounds>
```

Comment:
Unfortunately, i don't think that this content should be included in this patch.

Reply:
@lewismc I went ahead and merged master back into this feature branch so this is solved - see 1e2c84882b9d218c88752a93c4adaf167ae9355d